src/components/Main.js

```diff
@@ -68,7 +66,7 @@ export default class Main extends React.Component {
   render () {
     var scene = document.querySelector('a-scene');
```

Comment:
Just a suggestion that might not be relevant due to my lack of knowledge of react. Is it worth caching the `sceneEl` element instead of querying it each time we render?